modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ObjectFoldersSidebar.tsx

```diff
@@ -140,6 +140,19 @@ export default function ObjectFoldersSideBar({
 						key={currentObjectFolder.name}
 						onClick={() => {
 							setSelectedObjectFolder(currentObjectFolder);
+
+							const currentUrl = new URL(window.location.href);
```

Comment:
If you agree with the previous change, apply the same pattern here, `currentURL`